File: app/src/main/java/com/wineguesser/deductive/viewmodel/FinalConclusionFragmentViewModel.java
package com.wineguesser.deductive.viewmodel;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class FinalConclusionFragmentViewModel extends ViewModel {

    private final MutableLiveData<Boolean> isLoading = new MutableLiveData<>();

    public LiveData<Boolean> isLoading() {
        return isLoading;
    }

    public void setIsLoading(Boolean isLoading) {
        this.isLoading.setValue(isLoading);
    }
}

File: app/src/main/java/com/wineguesser/deductive/viewmodel/FinalConclusionFragmentViewModel.kt
package com.wineguesser.deductive.viewmodel

import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class FinalConclusionFragmentViewModel : ViewModel() {
    val isLoading = MutableLiveData<Boolean>()

    fun setIsLoading(isLoading: Boolean) {
        this.isLoading.value = isLoading
    }
}
